src/components/apis/list-of-apis/ko/runtime/api-list-dropdown.ts

```diff
@@ -86,7 +86,7 @@ export class ApiListDropdown {
             this.selectedApi(api);
         }
         catch (error) {
-            console.error(`Unable to load APIs. ${error}`);
+            throw error;
```

Comment:
Rupeng, please don't remove the message, because we'll need it to be logged by UnhandlerErrorHandler. So, it should be like this: throw new Error(`Unable to load APIs. ${error}`);